src/components/pages/contact/contact-form.tsx

```diff
@@ -38,21 +60,12 @@ export const ContactForm: React.FC = () => {
   } = useForm();
 
   const onSubmit = (formData: FormData) => {
-    // partialy taken from the Netlify Blog:
-    // - https://www.netlify.com/blog/2017/07/20/how-to-integrate-netlifys-form-handling-in-a-react-app/
-    const encodeForNetlify = (data: any): string => {
-      return Object.keys(data)
-        .map(
-          (key) =>
-            encodeURIComponent(key) + '=' + encodeURIComponent(data[key]),
-        )
-        .join('&');
-    };
```

Comment:
obsolete now as we can send json to the endpoint. I wonder why netlify did not allow that